sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/DataFlowRenameDataFlowOperation.cs

```diff
@@ -26,17 +26,20 @@ protected DataFlowRenameDataFlowOperation()
 
         internal DataFlowRenameDataFlowOperation(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, Request request, Response response)
         {
-            _operation = new OperationInternals(clientDiagnostics, pipeline, request, response, OperationFinalStateVia.Location, "DataFlowRenameDataFlowOperation");
+            var nextLinkOperation = NextLinkOperationImplementation.Create(pipeline, request.Method, request.Uri.ToUri(), response, OperationFinalStateVia.Location);
+            _operation = new OperationInternal(clientDiagnostics, nextLinkOperation, response, "DataFlowRenameDataFlowOperation");
         }
 
         /// <inheritdoc />
-        public override string Id => _operation.Id;
+#pragma warning disable CA1822
+        public override string Id => throw new NotImplementedException();
+#pragma warning restore CA1822
 
         /// <inheritdoc />
         public override bool HasCompleted => _operation.HasCompleted;
 
         /// <inheritdoc />
-        public override Response GetRawResponse() => _operation.GetRawResponse();
+        public override Response GetRawResponse() => _operation.RawResponse;
```

Comment:
Why not keep `GetRawResponse()` as the API here?

Reply:
`OperationInternal` and `OperationInternal<T>` have `RawResponse` property.